api/github.ts

```diff
@@ -454,29 +455,24 @@ async function repoStats ({topic, owner, repo, ...restArgs}: PathArgs) {
   }
 }
 
-function dependents (type: string) {
+function dependents (type: DependentsType) {
   return async function ({ owner, repo }: PathArgs) {
-    const html = await got(`https://github.com/${owner}/${repo}/network/dependents`, {
-      headers: {
-        Accept: 'text/html,application/xhtml+xml,application/xml'
-      }
-    }).text()
+    const html = await got(`https://github.com/${owner}/${repo}/network/dependents`).text()
+    const reDependents = new RegExp(`\\?dependent_type=${type}">(?:\\s*<svg.*<\\/svg>)?\\s*([\\d,]+)`)
+    const count = Number(html.match(reDependents)?.[1].replace(/,/g, ''))
+    const subject = type === 'PACKAGE' ? 'pkg dependents' : 'repo dependents'
 
+    if (Number.isNaN(count)) {
+      return {
+        subject,
+        status: 'invalid',
+        color: 'grey'
+      }
+    }
     return {
-      subject: type === 'PACKAGE' ? 'pkg dependents' : 'repo dependents',
-      status: parseDependents(html, type),
+      subject,
+      status: millify(count),
```

Comment:
`count` might be a `NaN`, which would cause `millify` throw an error

Reply:
True, I'll handle it explicitly.

Reply:
How I missed that block 🙈